In [3]:
import pandas as pd   
import numpy as np  
import matplotlib.pyplot as plt  
import seaborn as sns

4.	Using the pandas `read_csv()` method.
Read the GDP dataset into your notebook as a DataFrame called `gdp_df`.
Take a look at the first few and last few rows to familiarize yourself with what is contained in this dataset.

In [4]:
gdp_df = pd.read_csv('../data/gdp_percapita.csv')

In [7]:
gdp_df

Country or Area                                           Subgroup  \
0           Albania                                              Total   
1           Albania                                              Total   
2           Algeria                                              Total   
3           Algeria                                              Total   
4           Andorra                                              Total   
..              ...                                                ...   
342        Zimbabwe                                              Total   
343         fnSeqID                                           Footnote   
344               1  Data refer to years or periods other than thos...   
345               2  For a complete list of countries and territori...   
346               3  Due to the cession in July 2011 of the Republi...   

          Year         Source     Unit  Value  Value Footnotes  \
0    1990-2011  UN_World Bank  Percent    5.3              NaN   
1    1970-1990  UN_World Bank  Percent   -0.7              1.0   
2    1990-2011  UN_World Bank  Percent    1.5              NaN   
3    1970-1990  UN_World Bank  Percent    1.6              NaN   
4    1990-2011  UN_World Bank  Percent    2.5              1.0   
..         ...            ...      ...    ...              ...   
342  1970-1990  UN_World Bank  Percent   -0.4              NaN   
343        NaN            NaN      NaN    NaN              NaN   
344        NaN            NaN      NaN    NaN              NaN   
345        NaN            NaN      NaN    NaN              NaN   
346        NaN            NaN      NaN    NaN              NaN   

     Value Footnotes.1  
0                  NaN  
1                  1.0  
2                  NaN  
3                  NaN  
4                  1.0  
..                 ...  
342                NaN  
343                NaN  
344                NaN  
345                NaN  
346                NaN  

[347 rows x 8 columns]

5. How many rows and columns are in `gdp_df`?  347 rows.
What are the data types of each column? answer below.

In [11]:
print(gdp_df.dtypes)

Country or Area       object
Subgroup              object
Year                  object
Source                object
Unit                  object
Value                float64
Value Footnotes      float64
Value Footnotes.1    float64
dtype: object


6. Drop the `Value Footnotes` column and rename the remaining three to 'Country', 'Year', and 'GDP_Per_Capita'.